README.md

```diff
@@ -6,11 +6,58 @@
 
 CadenceBot is a [Discord](https://discordapp.com/) bot, which serves as a client for [Cadence Radio](https://github.com/kenellorando/cadence).
 
-CadenceBot is fully featured, and is functionally equivalent to the [web client](http://cadenceradio.com/).
+CadenceBot is fully featured, and is functionally equivalent to the [web client](http://cadenceradio.com/). You can add CadenceBot to your Discord server by clicking [this link.](https://discord.com/api/oauth2/authorize?client_id=372999377569972224&permissions=274881252352&scope=bot)
+
+## Implementation and details
```

Comment:
@kenellorando Do feel free to comment on my descriptions of the ecosystem here - I tried to be complete and correct while not going into more detail than is useful to understand the bot.